runtime/oti/j9nonbuilder.h

```diff
@@ -3082,6 +3082,7 @@ typedef struct J9ArrayClass {
 	struct J9Method** specialSplitMethodTable;
 	struct J9JITExceptionTable* jitMetaDataList;
 	struct J9Class* gcLink;
+	struct J9Class* hostClass;
```

Comment:
Temporarily added this in for compilation. Will remove the change once #7571 is merged in